lib/ui/dashboard.js

```diff
@@ -1,22 +1,24 @@
 'use strict';
 
-const blessed  = require('blessed'),
-      chart    = require('./chart'),
-      debounce = require('lodash.debounce'),
-      duration = require('moment-duration-format'),
-      get      = require('../http-then').get,
-      moment   = require('moment'),
-      spinner  = require('./spinner'),
-      task     = require('../task'),
-      util     = require('../util');
+const EventEmitter = require('events'),
+      blessed      = require('blessed'),
+      chart        = require('./chart'),
+      debounce     = require('lodash.debounce'),
+      duration     = require('moment-duration-format'),
+      get          = require('../http-then').get,
+      moment       = require('moment'),
+      spinner      = require('./spinner'),
+      task         = require('../task'),
+      util         = require('../util');
 
 const { isNotEmpty, humanBytes } = util;
 
 const { delay } = require('../promise');
 
-class Dashboard {
+class Dashboard extends EventEmitter {
```

Comment:
With the `tabs` logic encapsulated in a `menu` widget, we could avoid using events and just pass the menu instance to the dashboard to trigger the encapsulated logic in the `exec` widget.

Reply:
Done in 8303f6efb8337b1301194e88bfedb333e59afa9e.